emsdk.py

```diff
@@ -508,10 +508,8 @@ def file_to_lf(filename):
 # Removes a single file, suppressing exceptions on failure.
 def rmfile(filename):
   debug_print('rmfile(' + filename + ')')
-  try:
+  if os.path.exists(filename):
```

Comment:
does this work if filename is not a regular file?

Reply:
Hmm, you're right, lexists is a little safer

Reply:
or `isfile`?